src/appEntry.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from "react-dom";
+import { HashRouter, Route, Link } from "react-router-dom";
+import './appEntry.scss';
+// import './Routes.jsx';
+import App from './App.jsx';
+import 'bootstrap';
+
+const app = document.getElementById('app');
+ReactDOM.render(<HashRouter><App/></HashRouter>, app);
```

Comment:
`react-router-dom` might be overkill for our routing needs, especially since we're not using a full-blown browser with a navigation bar, we don't have a back button, and so on. This is somewhat subjective, and I'd argue in favor of keeping it simple, and using for example local component state to keep track of where we are in the application. I'm fine with merging this PR regardless of which path we take.